pyfftw/pyfftw.pyx

```diff
@@ -1394,6 +1405,10 @@ cdef class FFTW:
         copy the returned array.
         '''
 
+        if ortho and normalise_idft:
+            raise ValueError('Invalid options: ortho and normalise_idft cannot'
+                             ' both be True.')
+
```

Comment:
Agreed.  The alternative I considered was a norm kwarg that could be one of `{'ortho', 'idft', None}`, but that is not backwards compatible.